v2/controllers/networkplugins_controller.go

```diff
@@ -198,6 +198,12 @@ func (hostPlumberConfig *HostPlumberT) WriteConfigToTemplate(outputDir, registry
 		config["HostPlumberImage"] = ReplaceContainerRegistry(HostPlumberImage, registry)
 	}
 
+	if hostPlumberConfig.MetricsPort != "" {
+		config["MetricsPort"] = hostPlumberConfig.MetricsPort
+	} else {
+		config["MetricsPort"] = "8080"
```

Comment:
Use a constant for the port value here.

Reply:
@psarwate Sure!!!..I've changed in the recent commit